app/code/Magento/Catalog/Controller/Adminhtml/Product/Attribute/Save.php

```diff
@@ -327,8 +327,11 @@ private function preprocessOptionsData(&$data)
             $serializedOptions = json_decode($data['serialized_options'], JSON_OBJECT_AS_ARRAY);
             foreach ($serializedOptions as $serializedOption) {
                 $option = [];
-                $serializedOptionWithParsedAmpersand = str_replace('&', '%26', $serializedOption);
-                parse_str($serializedOptionWithParsedAmpersand, $option);
+                if (strpos($serializedOption, '=') !== false) {
+                    list($key, $value) = explode("=", $serializedOption);
+                    $serializedOption = implode("=", [$key, urlencode($value)]);
+                }
+                parse_str($serializedOption, $option);
```

Comment:
Please cherry pick corresponding commits from 2.3 instead of custom implementation: https://github.com/magento/magento2/blame/2.3-develop/app/code/Magento/Catalog/Controller/Adminhtml/Product/Attribute/Save.php#L144

Reply:
It's impossible, at least for me.